src/Adapter/MSTest.TestAdapter/Constants.cs

```diff
@@ -88,36 +88,6 @@ internal static class Constants
 
     internal static readonly TestProperty InnerResultsCountProperty = TestProperty.Register("InnerResultsCount", InnerResultsCountLabel, typeof(int), TestPropertyAttributes.Hidden, typeof(TestResult));
 
-    internal static readonly TestProperty TestRunIdProperty = TestProperty.Register(TestRunId, TestRunId, typeof(int), TestPropertyAttributes.Hidden, typeof(TestCase));
```

Comment:
We never set a value for any of these properties. @Evangelink Could they be magically set by something else (not us)? Or is it really dead code?